platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceImpl.java

```diff
@@ -850,28 +843,32 @@ private void uninstallAllFeatures(Application application) {
       Set<Feature> features = application.getFeatures();
       for (Feature feature : features) {
```

Comment:
✏️  How about a nice little streaming since you now moved the exception handling in that new `tryUninstallfeature()` method?